normandy/control/urls.py

```diff
@@ -0,0 +1,13 @@
+from django.conf.urls import url, include
+
+from normandy.control import views as control_views
+from django.contrib.auth.views import login, logout_then_login
+
+urlpatterns = [
+    url(r'^control/', include([
```

Comment:
We want the control interface to only be available on the admin node behind the VPN; we have a setting `ADMIN_ENABLED` to control this: https://github.com/mozilla/normandy/blob/master/normandy/urls.py#L15

This URLConf should only include URLs if the setting is true to avoid having it available outside the VPN.